packages/blade/src/components/Chip/ChipGroup.stories.tsx

```diff
@@ -0,0 +1,463 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import React from 'react';
+import { Heading, Text, Title } from '../Typography';
+import type { ChipGroupProps } from './ChipGroup';
+import { ChipGroup as ChipGroupComponent } from './ChipGroup';
+import { Chip as ChipComponent } from './Chip';
+import { PaymentLinksIcon, TagIcon, SmartphoneIcon } from '~components/Icons';
+import { Sandbox } from '~utils/storybook/Sandbox';
+import StoryPageWrapper from '~utils/storybook/StoryPageWrapper';
+import { Box } from '~components/Box';
+import { getStyledPropsArgTypes } from '~components/Box/BaseBox/storybookArgTypes';
+import iconMap from '~components/Icons/iconMap';
+import { Dropdown, DropdownButton, DropdownOverlay } from '~components/Dropdown';
+import { ActionList, ActionListItem } from '~components/ActionList';
+
+const Page = (): React.ReactElement => {
+  return (
+    <StoryPageWrapper
+      componentDescription="Chips represents a collection of selectable objects which enable users to make selections, filter content, and trigger relevant actions. Chips can have either single selection or multiple (based on context)."
+      componentName="ChipGroup"
+      imports={`import { Chip, ChipGroup } from '@razorpay/blade/components';\nimport type { ChipProps, ChipGroupProps } from '@razorpay/blade/components';`}
+      figmaURL={{
+        paymentTheme:
+          'https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=52377%3A23885&mode=design&t=y7gUIBIzzNMRd3w6-1',
+        bankingTheme:
+          'https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?type=design&node-id=18358%3A3135&mode=design&t=FzNrQV6ZZaLoxzcj-1',
+      }}
+    >
```

Comment:
put an alert above the Props table mentioning that these are just ChipGroup props and link to chip stories for props of chips